composer.json

```diff
@@ -5,36 +5,40 @@
     {
       "type": "vcs",
       "url": "https://github.com/silverstripe/silverstripe-globaltoolbar.git"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/sminnee/silverstripe-queuedjobs.git"
```

Comment:
Is there a blocker we need to get merged into queuedjobs first?

Reply:
Good point I'll have a look on hackday. This stuff may have already made its ways in.